.travis.yml

```diff
@@ -1,55 +1,62 @@
+sudo: required
+
 language: python
 
+dist: trusty
+
+notifications:
+    email: false
+
 python:
   - 2.7
   - 3.4
+  - 3.5
+
+addons:
+  apt:
+    packages: 
+      - gfortran
+      - liblensfun-dev
+      - libraw-dev
+      - libimage-exiftool-perl
+
 
 before_install:
   # Python
   - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
-  - chmod +x miniconda.sh
-  - ./miniconda.sh -b
-  - export PATH=/home/travis/miniconda2/bin:$PATH
-  - conda update --yes conda
-  
-  # system libs
-  - sudo apt-get install libraw-dev liblensfun-dev libnetcdf-dev libhdf5-serial-dev libimage-exiftool-perl gfortran
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH=$HOME/miniconda/bin:$PATH
+  - hash -r
   
+  # CDF
+  - wget -r -l1 -np -nd -nc http://cdaweb.gsfc.nasa.gov/pub/software/cdf/dist/latest-release/linux/ -A cdf*-dist-all.tar.gz
```

Comment:
actually `-nc` shouldn't be there, that says don't overwrite. Travis always has a fresh image for free tier, and in any case we would want to clobber with a new version if one existed.
Those options are to allow automatically getting the latest version without specifying version number.

-r recursive
-l1 down one level only
-np don't traverse up a directory
-nd don't save directory hierarchy